pennylane/grouping/graph_colouring.py

```diff
@@ -36,7 +36,17 @@ def largest_first(binary_observables, adj):
     """Performs graph colouring using the Largest Degree First heuristic. Runtime is quadratic in
     number of vertices.
 
-    **Usage example:**
+    Args:
+        binary_observables (array[bool]): the set of Pauli words represented by a column matrix
+            of the Pauli words in binary vector represenation
+        adj (array[bool]): the adjacency matrix of the Pauli graph
+
+    Returns:
+        dict(int, list[array[bool]]): keys correspond to colours (labelled by integers) and values
```

Comment:
From the examples below, this should be `float`. Although, it would probably make more sense for these to be int? Unless weighted adjacency matrices are allowed...
```suggestion
        binary_observables (array[float]): the set of Pauli words represented by a column matrix
            of the Pauli words in binary vector represenation
        adj (array[float]): the adjacency matrix of the Pauli graph

    Returns:
        dict(int, list[array[float]]): keys correspond to colours (labelled by integers) and values
```

Reply:
Thanks for catching that Josh! Think it could indeed only be int (binary for Paulis & adjacency without weights)